src/testinput/java/ps1/RatPoly.java

```diff
@@ -170,7 +170,7 @@ public String unparse() {
 
     StringBuffer sb = new StringBuffer();
     boolean isNegative;
-    RatNum coeff = null;
```

Comment:
The declaration should be moved into the for loop body, because there are no loop-carried dependences.